charts/influxdb2/values.yaml

```diff
@@ -120,6 +126,30 @@ persistence:
   mountPath: /var/lib/influxdb2
   subPath: ""
 
+## Add custom volume and volumeMounts
+##
+# volumes:
+#   - name: influxdb2-templates
+#     hostPath:
+#       path: /data/influxdb2-templates
+#       type: Directory
+# mountPoints:
+#   - name: influxdb2-templates
+#     mountPath: /influxdb2-templates
+#     readOnly: true
+
+## Allow executing custom init scripts
+## If the container finds any files with the .sh extension inside of the
+## /docker-entrypoint-initdb.d folder, it will execute them. The order they are
+## When multiple scripts are present, they will be executed in lexical sort order by name. For more details see Custom Initialization Scripts in https://hub.docker.com/_/influxdb .
+##
```

Comment:
```suggestion
## /docker-entrypoint-initdb.d folder, it will execute them. 
## When multiple scripts are present, they will be executed in lexical sort order by name.
## For more details see Custom Initialization Scripts in https://hub.docker.com/_/influxdb .
```